cmake/OpenCVFindLAPACK.cmake

```diff
@@ -106,11 +106,37 @@ macro(ocv_lapack_check)
       list(APPEND __link_directories ${LAPACK_LINK_LIBRARIES})
     endif()
 
+    set(LAPACK_TRY_COMPILE_DEF "")
+    if(LAPACK_IMPL STREQUAL "LAPACK/Apple") # https://github.com/opencv/opencv/issues/24660
+      # TODO: Need to find a way detecting macOS / iOS versions
+      #       enbale if macOS >= 13.3 or iOS >= 16.4
```

Comment:
to-do is not relevant any more.

Reply:
Done. I do not want to mess around with ios build script, so skipping this for ios.